lib/synapse/service_watcher/zookeeper_aurora.rb

```diff
@@ -0,0 +1,76 @@
+# encoding: utf-8
+#
+# Variant of the Zookeeper service watcher that works with Apache Aurora
+# service announcements.
+#
+# Parameters:
+#   hosts: list of zookeeper hosts to query (List of Strings, required)
+#   path: "/path/to/serverset/in/zookeeper" (String, required)
+#   port_name: Named service endpoint (String, optional)
```

Comment:
Can we call this 'endpoint_name'? I'm a little confused by 'port_name' what it actually does